charts/rs-server-cadip/templates/configmap.yaml

```diff
@@ -102,17 +102,28 @@ data:
             next_page_url_tpl: '{url}?{search}&$top={items_per_page}'
             total_items_nb_key_path: '$.total'
 
+          sort:
+            sort_by_tpl: '&$orderby={sort_param} {sort_order}'
+            sort_param_mapping:
+              startTimeFromAscendingNode: DownlinkStart
```

Comment:
Same here @vprivat-ads, if I look at your two last changes in rs-server cadip config files: 
1. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config.yaml
2. https://github.com/RS-PYTHON/rs-server/blob/d682b21d14402554c9320481c58929faacc34e17/services/cadip/config/cadip_ws_config_token_module.yaml

The 3 fields: 
```yaml
        startTimeFromAscendingNode: DownlinkStart
        completionTimeFromAscendingNode: DownlinkStop
        publicationDate: PublicationDate
```

are defined only for the _session stations. But in this helm chart, you defined them for all stations.

Reply:
This needs to be updated according to https://github.com/RS-PYTHON/rs-server/blob/b6fd7ea4c4939b80fe862b8b8bc440eb3be54653/services/cadip/config/cadip_ws_config_token_module.template.yaml#L92

Reply:
I think for the "non session" stations, only publicationDate should be added, there is no downlinkStart/Stop info at file level, it's only at session level